Графика для всех страниц

eldarnugaev opened the pull request:
---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/298013/tasks/7)

Commit:
Добавляет файлы с графикой в проект

Commit:
Внедряет графику в разметку главной страницы

Commit:
Заменяет разметку разделителя в блоке преимуществ на главной странице

Изменена реализацию: вместо разделителя отдельным изображением теперь используется тег hr.

Commit:
Внедряет графику в разметку страницы каталога

Commit:
Изменена разметка отображения рейтинга на странице каталога

boumrz commented on index.html:
```diff
@@ -6,68 +6,72 @@
         <ul class="navigation-list navigation-user">
-          <li class="navigation-item"><a class="navigation-link" href="index.html"><img src="" alt="Поиск"></a></li>
-          <li class="navigation-item"><a class="navigation-link" href="index.html"><img src="" alt="Избранное"></a></li>
+          <li class="navigation-item"><a class="navigation-link" href="index.html"><img src="images/icon-usermenu-search.svg" width="20" height="20" alt="Поиск"></a></li>
```
Я боюсь, что если указать путь до ресурса без слеша ('/'), то на стенде (когда будешь заходить на свой сайт) не будет отображаться твоя картинка, потому что нужно все изображения брать от корня

http://htmlbook.ru/samhtml/izobrazheniya/dobavlenie-risunka

boumrz commented on index.html:
```diff
@@ -6,68 +6,72 @@
       <section class="advantage high-accent">
         <h3>Настоящий<br>городок</h3>
-        <img class="advantage-devider" src="#" alt="Разделитель перед объяснением">
+        <hr class="devider-invert" width="60" size="2" noshade>
```
А вам так в лайве показывали? Эту линию лучше делать стилями, а не разметкой

eldarnugaev replied:
Заменил на div с классом для последующей стилизации.

Commit:
Добавляет исправления по замечаниям для главной страницы

Добавляет слеш ('/') перед указанием пути к файлу с графикой.
Изменяет разметку разделителя: вместо hr используется div с классом для последующей стилизации.

Commit:
Добавляет исправления по замечаниям к странице каталога

Добавляет слеш ('/') к пути графического файла.

boumrz commented on catalog.html:
```diff
@@ -6,16 +6,16 @@
   <body>
     <header class="page-header">
-      <img class="logo-sedona" src="#" alt="Логотип проекта Седона">
+      <img class="logo-sedona" src="/images/logo-header-sedona.svg" width="140" height="70" alt="Логотип проекта Седона">
```
Используя слеш вначале может возникнуть проблема при загрузке картинки на сайте, потому что этот путь будет подставляться к корню сайта

boumrz reviewed (approved):
Есть вопросы по относительным путям, но в целом можно мержить эту прку.
